htdocs/sass/rcloud-discover.scss

```diff
@@ -99,6 +99,10 @@ progress::-webkit-progress-value {
 .grid-item h1{
     font-size: 17px;
     margin: 0;
+    /* firefox */
+    white-space: pre-wrap;
+    word-break: break-all;
+    /* chrome */
     word-break: break-word;
```

Comment:
I've checked the visual output in Chrome and Firefox on Ubuntu, and it is correct. Apologies if I've misinterpreted your _verify and update_ statement.